pom.xml

```diff
@@ -47,22 +53,22 @@
     <tag>${scmTag}</tag>
   </scm>
 
-  <licenses>
-    <license>
-      <name>MIT License</name>
-      <url>http://opensource.org/licenses/MIT</url>
-    </license>
-  </licenses>
-
   <properties>
-    <revision>1.29</revision>
+    <revision>1.30</revision>
```

Comment:
Why skipping a version?

Reply:
This is unnecessary and should be reverted.

Reply:
I bumped it because a bunch of deps were updated in `pom.xml`. I'm not sure if it's a strong reason to increment the version number, and I don't have a strong preference either way.

Reply:
Whenever the next version of the plugin is released, it will have a new version anyway. You are just going from 1.28 to 1.30 and skipping 1.29.

Reply:
I'll go ahead and revert it so that `mvn:release` can deal with it 😄

Reply:
https://github.com/jenkinsci/aws-credentials-plugin/pull/77/commits/c8eb71275348ce5c28ee24ae553c9e5cc51fd8ca